### ISSUE TITLE
Upload of main image doesn't work (despite resize)

### ISSUE THREAD

**Issue by kathakloss** [cafebabel/cafebabel.com#281]
Page: article/edit

Quand j'essaye d'uploader une image (à priori de la bonne taille 447 Ko, déjà recroppée) le système ne réagit pas https://preprod.cafebabel.com/fr/article/draft/5a9d7907f723b33c0d329d84/

Voici l'image:
![image](https://user-images.githubusercontent.com/37069979/36988636-a1cd9a92-209f-11e8-9b39-2527b9ed2475.png)

**Comment by vinyll** [comment]
Effectivement, lorsque je sélectionne l'image ou que je la glisse, elle n'apparait pas dans la page.
Tu peux jeter un oeil @cafebabel-webmaster ?

**Comment by johangiraud** [comment]
Oui ça m'a fait ça aussi avec ce fichier : 
![27f3816aed817cb844a51012de66c8fa](https://user-images.githubusercontent.com/32518627/37021950-ada24e76-2121-11e8-8637-b451f72d5d90.jpg)
alors que la taille est de 387Ko :/ Je ne comprends vraiment pas pourquoi. J'ai dû l'ouvrir sous photoshop, le réexporter et là ça fonctionnait avec le nouveau fichier exporté. Un idée @vinyll ?
Après il faut soit modifier les recommandations quant à la taille max de fichier (pour l'instant c'est encore écrit max 1Mo) soit modifier la taille maximum acceptée. Mais je ne sais pas trop comment faire. En fait dans le js il faut faire une condition qui dit que lorsqu'on est sur `profile/edit`, max-size=500Ko et quand on est sur `article/edit`, max-size=1Mo. Tu sais comment faire facilement @vinyll ?
